src/docfx/docset/Document.cs

```diff
@@ -456,8 +456,8 @@ private static bool IsValidRelativePath(string path)
                 ? sourcePath
                 : mappedSourcePath;
 
-            // if source ends with index.yml, change it to index.md
-            if ("index.yml".Equals(Path.GetFileName(sourcePath).ToLowerInvariant()))
+            // if source is landing page and not resource, change it to *.md
+            if (Schema.Type == typeof(LandingData) && ContentType != ContentType.Resource)
```

Comment:
do we need extra `ContentType != ContentType.Resource` ?

Reply:
you are right, we don't need it